src/simple/Auth.js

```diff
@@ -152,15 +148,27 @@ class Auth {
 
     setApiToken(session.token);
 
-    session.profile = await getApiClient().confd.getUser(session.uuid);
+    const profile = await getApiClient().confd.getUser(session.uuid);
```

Comment:
Can we put `await getApiClient().confd.getUser` and `await getApiClient().confd.getInfos()` in a `Promise.all()` ?

Reply:
Good idea.